frontend/src/pages/index.js

```diff
@@ -2,12 +2,17 @@ import { useKeycloak } from '@react-keycloak/web';
 import React from 'react';
 import EligibilityPageContainer from '../components/Eligibility/EligibilityPageContainer';
 import Layout from '../components/Layout';
+import {Helmet} from "react-helmet";
 
 function Index() {
   const { keycloak } = useKeycloak();
 
   return (
     <div className="app">
+       <Helmet>
+            <meta charSet="utf-8" />
+            <title>Passenger vehicle rebate log in – CleanBC Go Electric</title>
+        </Helmet>
```

Comment:
There are a few more titles in the ticket that have to be done:

Passenger vehicle rebate log in – CleanBC Go Electric
Passenger vehicle rebate application form – CleanBC Go Electric
Passenger vehicle rebate Identification requirements – CleanBC Go Electric
Passenger vehicle rebate application confirmation – CleanBC Go Electric

Reply:
added new titles